torch/testing/_internal/common_distributed.py

```diff
@@ -68,17 +68,14 @@ def wrapper(*args, **kwargs):
 
 def skip_if_not_multigpu(func):
     """Multi-GPU tests requires at least 2 GPUS. Skip if this is not met."""
-    def decorator(func):
-        @wraps(func)
-        def wrapper(*args, **kwargs):
-            if torch.cuda.is_available() and torch.cuda.device_count() >= 2:
-                return func(*args, **kwargs)
-            message = "Need at least {} CUDA devices".format(2)
-            TEST_SKIPS["multi-gpu"] = TestSkip(75, message)
-            sys.exit(TEST_SKIPS['multi-gpu'].exit_code)
-        return wrapper
-
-    return decorator
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        if torch.cuda.is_available() and torch.cuda.device_count() >= 2:
```

Comment:
is this the same as `@skip_if_lt_x_gpu(2)`? If so, will it work if we consolidate all `@skip_if_not_multigpu` occurrences into  `@skip_if_lt_x_gpu(2)`?

Reply:
Yep, great point. I was working on just removing this entirely and replacing all references to `@skip_if_not_multigpu` with `skip_if_lit_x_gpu(2)`, just need to figure out the failures for the tests that are now reenabled.